infra/feast-operator/internal/controller/services/services.go

```diff
@@ -358,6 +358,11 @@ func (feast *FeastServices) getContainerCommand(feastType FeastServiceType) []st
 	deploySettings := FeastServiceConstants[feastType]
 	targetPort := deploySettings.TargetHttpPort
 	tls := feast.getTlsConfigs(feastType)
+	logLevel := feast.getLogLevelForType(feastType)
+	if logLevel != nil {
+		deploySettings.Command = append(deploySettings.Command, "--log-level="+strings.ToUpper(*logLevel))
+	}
+
```

Comment:
its a nit but lets not split up the tls logic here -
```suggestion
	logLevel := feast.getLogLevelForType(feastType)
	if logLevel != nil {
		deploySettings.Command = append(deploySettings.Command, "--log-level="+strings.ToUpper(*logLevel))
	}
	tls := feast.getTlsConfigs(feastType)
```